table/tables/tables.go

```diff
@@ -540,6 +542,18 @@ func (t *TableCommon) rebuildIndices(ctx sessionctx.Context, txn kv.Transaction,
 	return nil
 }
 
+func appendMetaCol(sctx sessionctx.Context, colIDs []int64, row []types.Datum) ([]int64, []types.Datum) {
+	if shouldWriteMetaCol(sctx) {
+		colIDs = append(colIDs, model.ExtraMetaColID)
+		row = append(row, types.NewMetaDatum(sctx.GetSessionVars().TiDBWriteByTiCDC))
```

Comment:
```suggestion
		row = append(row, types.NewMetaDatum(sctx.GetSessionVars().TiDBWriteByTiCDC))
```

`sctx.GetSessionVars().TiDBWriteByTiCDC` is always true.

Reply:
done